projects/utils/src/api/editor/api.ts

```diff
@@ -1,7 +1,8 @@
 import { HighlightPosition } from './type'
+import { StatusEvents } from '../../types'
 
 export interface IEditor {
-  events: {}
+  events: StatusEvents
```

Comment:
Tiny comment, for better readability perhaps we should always write `{} & StatusEvent`
If no other event are described. Also I suppose `StatusEvent` should be singular.